test/feature-erc20.js

```diff
@@ -140,7 +141,12 @@ describe('Feature ERC20', function () {
 
     // Wait until the transaction is mined
     const transactionMinedClaimTx = await claimTx.wait();
+    const txID = transactionMinedClaimTx.events[0].args[0].toString();
 
+    // Get the "proof" from the claim.
+    const proofExpected = (await contractAsSignerReceiver1.claims(txID))[3];
+
+    expect(proofExpected).to.match(/^https:\/\/github.com\//g);
```

Comment:
Why do you use a regex ? Just use `expect(proofExpected).to.equal('https://github.com/feature-sh/bot/pull/190');`